metron-platform/metron-elasticsearch/src/test/java/org/apache/metron/elasticsearch/integration/components/ElasticSearchComponent.java

```diff
@@ -277,7 +277,7 @@ public boolean hasIndex(String indexName) {
     @Override
     public void stop() {
       try {
-        node.close();
+          node.close();
```

Comment:
This is not needed.  Please remove this.

Reply:
Done @nickwallen